tests/test_build_html.py

```diff
@@ -451,15 +451,24 @@ def test_html_download(app):
 @pytest.mark.sphinx('html', testroot='roles-download')
 def test_html_download_role(app, status, warning):
     app.build()
-    digest = md5((app.srcdir / 'dummy.dat').encode()).hexdigest()
+    digest = md5(b'dummy.dat').hexdigest()
     assert (app.outdir / '_downloads' / digest / 'dummy.dat').exists()
+    assert not (app.outdir / '_downloads' / digest / 'dummy.dat').text()
```

Comment:
What this line confirms? I feel this is meaningless.

Reply:
One of the `dummy.dat` files is empty, another one (which I added) has some text.

These lines make sure the proper files are picked — empty in the first case, non-empty in the second one.

Reply:
I have added an explanatory comment now.

Reply:
Yes, I understand that. But why do we need to check the contents of `dummy.dat`?
Is it important to check the file is empty?

If you'd like to confirm copying goes well, this is not good testing. I think 1) filling the some content to the file, and 2) comparing the source and destination files is better.

Reply:
I wanted to make sure that in the right file (of two) is copied, not that the copying goes well. But OK, I will remove these checks now.